hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/ReplaceArchivalHelper.java

```diff
@@ -68,6 +68,11 @@
   public static boolean deleteReplacedFileGroups(HoodieEngineContext context, HoodieTableMetaClient metaClient,
                                                  TableFileSystemView fileSystemView,
                                                  HoodieInstant instant, List<String> replacedPartitions) {
+    // There is no file id to be replaced in the very first replace commit file for insert overwrite operation
+    if (replacedPartitions.isEmpty()) {
+      LOG.warn("Found empty partitionToReplaceFileIds");
```

Comment:
can use  partitionToReplaceFileIds -> replacedPartitions better?

Reply:
`partitionToReplaceFileIds` is the field name directly taken from the replace commit file. I guess it is subject to change. What about just explaining the warning as `Found no partition files to replace`?

Reply:
yes, "Found no partition files to replace " will better